test/run_test.py

```diff
@@ -165,6 +165,20 @@ def skip_test_p(name: str) -> bool:
     "test_typing",
     "distributed/elastic/events/lib_test",
     "distributed/elastic/agent/server/test/api_test",
+    "distributed/fsdp/test_utils",
+    "distributed/fsdp/test_reduce_scatter_bucketer",
+    "distributed/fsdp/test_fsdp_uneven",
+    "distributed/fsdp/test_fsdp",
+    "distributed/fsdp/test_fsdp_summon_full_params",
+    "distributed/fsdp/test_fsdp_state_dict",
+    "distributed/fsdp/test_fsdp_overlap",
+    "distributed/fsdp/test_fsdp_multiple_wrapping",
+    "distributed/fsdp/test_fsdp_multiple_forward",
+    "distributed/fsdp/test_fsdp_memory",
+    "distributed/fsdp/test_fsdp_input",
+    "distributed/fsdp/test_fsdp_freezing_weights",
+    "distributed/fsdp/test_fsdp_apply",
+    "distributed/fsdp/test_flatten_params_wrapper",
```

Comment:
To avoid duplication, can we perhaps create `DISTRIBTUED_FSDP_TESTS=[....]` and then append it to the 3 blocklists, rather than duplicating the same thrice. Also, consider using glob of `discover_tests` routine rather than listing all the test names by hand

Reply:
sounds good, will change it